jenkins.yml

```diff
@@ -0,0 +1,10 @@
+project:
+  name: kitsune
+  docker_name: mozorg/kitsune
```

Comment:
We have a `mozmeao` docker org as well that I think is the place for things in future. The "mozorg" org is primarly for bedrock and I think bedrock should move as well. Not a blocker, just something I think we should eventually do.

Reply:
Updated